Makefile

```diff
@@ -23,10 +23,10 @@ static-flake8:
 	@find scripts -type f -not -path "scripts/qesap/.tox/*" -not -path "scripts/qesap/.venv/*" -name \*.py -exec flake8 --ignore=E501 {} +
 
 static-ansible-yaml:
-	@find ansible -type f -iname "*.yaml" -or -iname "*.yaml" -exec yamllint {} +
+	@tools/ansible_yaml_lint
 
 static-ansible-syntax:
-	@find ansible/playbooks -type f -iname "*.yaml" -maxdepth 1  -exec ansible-playbook -l all -vvvv -i terraform/azure/inventory.yaml --syntax-check  {} +
+	@find ansible/playbooks/ -type f -iname "*.yaml" -and ! -name "registration_role.yaml" -maxdepth 1  -exec ansible-playbook -l all -vvvv -i terraform/azure/inventory.yaml --syntax-check  {} +
```

Comment:
The other check (the one in `static-ansible-yaml`) calls a script with only one `find` (and some variables). Wouldn't it make sense to also turn this check into a script in `tools/`?

Reply:
I decided to focus this PR on yamllint and leave all the work about `ansible-playbook ``syntax-check` for a next PR. Yes I agree it make sense to move this one too to a dedicated script.